src/main/java/ca/bc/gov/hlth/pbfdataloader/service/PGPService.java

```diff
@@ -45,4 +45,11 @@ public File decrypt(File encryptedFile) {
 		}
 	}
 	
+	private File generateTempFile(File encryptedFile) throws IllegalArgumentException, IOException {
+		// This will strip the .gpg extension from foo.zip.gpg leaving foo.zip
+		String decryptedFileName = FilenameUtils.getBaseName(encryptedFile.getName());
+		
+	    return File.createTempFile(FilenameUtils.getBaseName(decryptedFileName), "." + FilenameUtils.getExtension(decryptedFileName));
```

Comment:
Could a SEPARATOR constant be  used for "." like in SFTPService for conformity. Or is it OK as it's only used once?

Reply:
Yes, it should. Just got lazy while refactoring.